README.adoc

```diff
@@ -165,9 +140,9 @@ You will look into the `InventoryManager` class in more detail in the next secti
 
 === Adding the @Fallback annotation
 
-The `inventory` microservice is now able to recognize that the `system` microservice
-was taken down for maintenance because of the 503 HTTP response.
-This method throws an IOException to simulate the system being unavailable.
+The `inventory` service is now able to recognize that the `system` service
+was taken down for maintenance.
+An IOException is thrown to simulate the `system` service being unavailable.
```

Comment:
What about, "...to simulate that the `system` service is unavailable."

Reply:
Fixed